models/pipeline.js

```diff
@@ -62,7 +67,7 @@ module.exports = {
      * @type {Joi}
      */
     get: Joi.object(mutate(MODEL, [
-        'id', 'scmUri', 'createTime', 'admins'
+        'id', 'scmUri', 'scmContext', 'createTime', 'admins'
```

Comment:
Should we wait till the sequelize problem is fixed and then merge this? I think they want the scmContext to be required.